extensions/2.0/Khronos/KHR_materials_sheen/README.md

```diff
@@ -0,0 +1,226 @@
+
+If you want to trade a bit of accuracy for more performance, you can use the `VdotN` term only and thus avoid doing multiple lookups for `LdotN`. The albedo scaling term is simplified to:
+```glsl
+albedoScaling = 1.0 - max3(sheenColor) * E(VdotN)
+```
+
+In this simplified form, it can be used to scale the base layer for both direct and indirect lights:
+```glsl
+specular_direct *= albedoScaling;
+diffuse_direct *= albedoScaling;
+environmentIrradiance_indirect *= albedoScaling
+specularEnvironmentReflectance_indirect *= albedoScaling
+```
+
+## Reference
+
+### Theory, Documentation and Implementations
+
+[Filament Material models - Cloth model](https://google.github.io/filament/Materials.md.html#materialmodels/clothmodel)
+
+[cmgen tool from Filament](https://github.com/google/filament)
+
+[NP13 David Neubelt, Matt Pettineo – “Crafting a Next-Gen Material Pipeline for The Order: 1886”, SIGGRAPH 2013](https://blog.selfshadow.com/publications/s2013-shading-course/rad/s2013_pbs_rad_notes.pdf)
+
+[EK17 Alejandro Conty Estevez, Christopher Kulla – “Production Friendly Microfacet Sheen BRDF”, SIGGRAPH 2017](https://blog.selfshadow.com/publications/s2017-shading-course/imageworks/s2017_pbs_imageworks_sheen.pdf)
+
+[AS07 Michael Ashikhmin, Simon Premoze – “Distribution-based BRDFs”, 2007](http://www.cs.utah.edu/~premoze/dbrdf/dBRDF.pdf)
+
+[cloth-shading](https://knarkowicz.wordpress.com/2018/01/04/cloth-shading/)
+
+[Enterprise PBR Shading Model - Sheen](https://dassaultsystemes-technology.github.io/EnterprisePBRShadingModel/spec-2021x.md.html#components/sheen)
+
+## Appendix: Full Khronos Copyright Statement
+
+Copyright 2018-2020 The Khronos Group Inc.
+
+Some parts of this Specification are purely informative and do not define requirements
+necessary for compliance and so are outside the Scope of this Specification. These
+parts of the Specification are marked as being non-normative, or identified as
+**Implementation Notes**.
+
+Where this Specification includes normative references to external documents, only the
+specifically identified sections and functionality of those external documents are in
+Scope. Requirements defined by external documents not created by Khronos may contain
+contributions from non-members of Khronos not covered by the Khronos Intellectual
+Property Rights Policy.
+
+In particular, the referenced paragraphs of ISO 16684-1:2019 are normative and so are
+INCLUDED in the Scope of this Specification and may contain contributions from non-members
+of Khronos that are not covered by the Khronos Intellectual Property Rights Policy.
+The links to https://github.com/adobe/xmp-docs are purely informative and so are EXCLUDED
```

Comment:
Totally my fault, I knew XMP had updated trademarks, but I didn't realize XMP had its own paragraph snuck in here.  Please remove this paragraph from Sheen, from lines 183 to 189 in this revision.  The result should match the new version in clearcoat in #1836.